zxlive/graphscene.py

```diff
@@ -195,6 +195,20 @@ def select_all(self) -> None:
         for it in self.items():
             it.setSelected(True)
 
+    def shutdown(self):
+
+        """
+        Perform necessary cleanup to avoid RunTimeError when the application closes.
+        Call this method before the application closes. 
+        """        
+
+        #Clear all items from the scene to ensure no dangling references remain
+        self.clear()
+
```

Comment:
Do you mind cleaning up some extra empty lines in this function?

Reply:
Hi, @RazinShaikh I have made the suggested changes, here is the [commit](https://github.com/Quantomatic/zxlive/pull/230/commits/cb113bde6d0ba31fad648426a89c990a0b4a8d2f)